meson.build

```diff
@@ -1061,7 +1061,8 @@ threads = dependency('threads')
 librt = cc.find_library('rt')
 libm = cc.find_library('m')
 libdl = cc.find_library('dl')
-libcrypt = cc.find_library('crypt')
+libcrypt = dependency('libcrypt')
+libcap = dependency('libcap')
```

Comment:
fallbacks are missing now?

Reply:
please mention that in the commit message (ideally with the versions that introduced the .pc files) as it wasn't clear, it just says `meson: first try dependency(), then fallback to find_library()`

Reply:
what about libcrypt and libacl?

Reply:
Oops. The same. Will update the comment.